source/developer/mobile-developer-setup.rst

```diff
@@ -117,10 +188,128 @@ Android (Device)
+
+Every make command has to be run from a terminal in the project's root directory.
+
+Commands to Prepare the App
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+These make commands are used to install dependencies, to configure necessary steps before running or building the app, and to clean everything.
+
+ - **make pre-run**: Downloads any project dependencies needed and sets up the app assets. This is the equivalent of running  ``.yarninstall``, ``.podinstall`` and ``dist/assets`` make commands.
+ - **make .yarninstall**: Downloads JavaScript and react native dependencies. Once this command finishes executing it will create a ``.yarninstall`` empty file in the project's root directory to indicate that this command already ran. If yarn updates your local dependencies in the ``node_modules`` directory, it will automatically run ``make post-install`` for you.
+ - **make .podinstall**: Downloads cocoapods dependencies needed to build the iOS app. Once this command finishes executing it will create a ``.podinstall`` empty file in the project's root directory to indicate that this command already ran.
+ - **make post-install**: Normally this command runs automatically if yarn detects that the project updated some or all of its JavaScript dependencies. Once yarn finishes executing, this command automatically sets up a few post installation steps to ensure that everything runs correctly.
+ - **make clean**: Removes all downloaded dependencies, clears the cache of those dependencies and deletes any builds that were created. It will not reset the repo, so your current changes will still be there.
+ - **make dist/assets**: Builds the assets to be used including images, localization files and overriding of asset files (see Override Assets & White Labeling for details).
+ 
+Commands to Run the App
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+These make commands are used to run the app on a device or emulator for Android, and on a simulator for iOS. (see ``Running the App on a Device`` for details).
+
+ - **make start-packager**: Runs the react-native packager used to bundle the JavaScript code. This command will execute *pre-run* to ensure the app is prepared.
+ - **make start**: Alias of ``make start-packager``.
+ - **make stop-packager**: Stops the react-native packager if it is running.
+ - **make stop**: Alias of ``make stop-packager``.
+ - **make run-ios**: Compiles and runs the app for iOS on an iPhone 6 simulator by default. You can set the environment variable SIMULATOR to the name of the device you want to use. This command will execute ``make start`` to ensure the packager is running.
+ - **make run**: Alias of ``make run-ios``.
+ - **make run-android**: Compiles and runs the app for Android on a running emulator or a device connected through USB. This command will execute ``make start`` to ensure the packager is running (see Create and Manage Virtual Devices to configure and run the Android emulator).
```

Comment:
Do we have a link for `Create and Manage Virtual Devices to configure and run the Android emulator`

Reply:
The link is https://developer.android.com/studio/run/managing-avds.html